testplan/web_ui/testing/src/Report/BatchReportBeta/components/NavBreadcrumbWithNextRoute.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Route } from 'react-router';
+import connect from 'react-redux/es/connect/connect';
+import _isString from 'lodash/isString';
+import _isObject from 'lodash/isObject';
+import _isArray from 'lodash/isArray';
+import uriComponentCodec from '../../../Common/uriComponentCodec';
+import { setHashComponentAlias } from '../state/uiActions';
+import { mkGetUIHashComponentToAlias } from '../state/uiSelectors';
+import NavBreadcrumb from './NavBreadcrumb';
+
```

Comment:
The usage of connect call is not easy to read. It will helpful if mapStateToProps / mapDispatchToProps / mergeProps are first separately defined.

Reply:
will do